Readme.md

```diff
@@ -139,12 +139,12 @@ var Pixel = Color.extend({
 
 var pixel = Pixel.new(11, 23, 'CC3399')
 pixel.toString()              // 11:23@#CC3399
-Pixel.isPrototypeOf(pixel)
+Pixel.isPrototypeOf(pixel)    // true
 
 // Pixel instances inhertis from `Color`
 Color.isPrototypeOf(pixel)    // true
 
 // In fact `Pixel` itself inherits from `Color`, remember just simple and
-// pure prototypal inheritance where object inherit from objects.
-Color.isPrototypeOf(Pixel)
+// pure prototypal inheritance where object inherits from objects.
```

Comment:
It's actually `objects inherit from objects` ([wiki](http://en.wikipedia.org/wiki/Prototype-based_programming))